scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -1661,19 +1692,14 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
 
   def caseClause(): Case = atPos(in.prevTokenPos, auto) {
     require(token.isNot[`case`] && debug(token))
-    token match {
-      case token: Unquote if token.prev.is[Ellipsis] =>
-        unquote[Case]
```

Comment:
I'm trying to push unification, because then we'll have less places to change when time will come to make adjustments to our quasiquotes (e.g. when complex ellipses become a thing).